src/RSocketClient.cpp

```diff
@@ -41,13 +42,22 @@ folly::Future<std::unique_ptr<RSocketRequester>> RSocketClient::connect(
     keepaliveTimer = std::move(keepaliveTimer),
     stats = std::move(stats),
     networkStats = std::move(networkStats),
-    promise = std::move(promise)](
-      std::unique_ptr<DuplexConnection> connection,
-      folly::EventBase& eventBase) mutable {
+    promise = std::move(promise)
+  ](std::unique_ptr<DuplexConnection> connection,
+    bool isFramedConnection,
+    folly::EventBase& eventBase) mutable {
     VLOG(3) << "onConnect received DuplexConnection";
 
+    std::unique_ptr<DuplexConnection> framedConnection;
```

Comment:
lets move this block of code including the isFramedConnection parameter to RSocketClient::fromConnection method because it is the counterpart of the TcpConnectionFactory::createDuplexConnectionFromSocket used by the RTGW.